### ISSUE TITLE
Admin Theme - ContentPicker - Display and functionality issues

### ISSUE THREAD

**Issue by rtpHarry** [OrchardCMS/Orchard#7263]
Navigation > Menu > Content Menu Item > Browse

![image](https://cloud.githubusercontent.com/assets/1038062/19212863/c847e208-8d53-11e6-8353-1ab6a7e381ab.png)
1. Select button should be right aligned:
      ![image](https://cloud.githubusercontent.com/assets/1038062/19212874/28926160-8d54-11e6-873a-e4bf1054d6f0.png)
2. Spacing between last item and button
3. All buttons need icons
4. Pager styles are wrong:
      ![image](https://cloud.githubusercontent.com/assets/1038062/19212885/5c048208-8d54-11e6-886d-d86247952901.png)
5. Select single and Select Multiple UI both showing on a select single picker

**Comment by petedavis** [comment]
Item 1 is because of this.

```
/* from orchard-content-items.less */
            @media (max-width: @screen-xs-max) {
                flex-direction: column-reverse;
            }
```

If the window was a few pixels wider by default it would appear as below. Is there another area that the classes `.content-items .summary .properties` is used? Looks like it could be removed if this is the only place that it applies?

![image](https://cloud.githubusercontent.com/assets/164331/19555519/a7d156b6-9704-11e6-8475-ab3565999804.png)

This is with that media selector removed:
![image](https://cloud.githubusercontent.com/assets/164331/19555656/4d8107f0-9705-11e6-981e-49856e8ec565.png)

**Comment by petedavis** [comment]
Just another not, is there an icon selected for use on a select button? The same icon will need to be used on the taxonomies parent term selector page:
![image](https://cloud.githubusercontent.com/assets/164331/19556112/2e255440-9707-11e6-9fc7-b0120da60b62.png)

**Comment by rtpHarry** [comment]
@petedavis good work, if you want to carry on with this one and make a PR please go ahead, I have marked it assigned to me but I haven't begun any work on it yet.

The buttons do want icons. I have done some work in this area, I have notes ready for a full review to bring all of the icons in line. I put some initial docs [here](https://github.com/OrchardCMS/OrchardDoc/issues/311). I picked one common icon for each action based on usage already in place in the admin theme repo. I also have a big list of notes not published with buttons that use variations or non-standard icons which I hope to bring them all in line before the theme reaches dev branch. So it looks like they should have:

    <i class="fa fa-check" aria-hidden="true"></i> @T("Select")

**Comment by petedavis** [comment]
No probs I will fix a few of them. Also done some rearranging so the admin theme font and colors are used. It was also overwriting the font awesome font-family on icon elements. Page could probably do with a restructure to use a responsive grid layout rather than the current fixed width of 640px.

Currently there is no suggestion for "Apply" in the list. Check feels appropriate, but would clash with select?

Also I have reduced the font size of the metadata to try and ensure it is always in a single line.

![image](https://cloud.githubusercontent.com/assets/164331/19586656/bf2c41a0-979e-11e6-8b84-32898d04bc80.png)

**Comment by Skrypt** [comment]
fixed by 5c519a1a045ffefc9fd0ba09f5a39a6cc28b0719
I reapplied @petedavis PR on a commit and made some tweaks on it just for the records.
